### PR TITLE
Rename None aggregation to Drop

### PR DESCRIPTION
**Pull request by cijothomas**
Drop feels like a better name than "none". Want to check if others feel the same.

"The None Aggregation informs the SDK to ignore/**drop** all Instrument Measurements for this Aggregation."

Found some history: https://github.com/open-telemetry/opentelemetry-specification/pull/956/files

### CODE REVIEW

**Review by alanwest — APPROVED** [review]

**Comment by jsuereth** [comment]
From one point of view I agree with this, WDYT of this change in relation to: https://github.com/open-telemetry/opentelemetry-specification/pull/2102

**Review by reyang — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Comment by reyang** [comment]
We've discussed this during the [11/11/2021 Metrics SIG Meeting](https://docs.google.com/document/d/1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo/edit#heading=h.gjr4yo24h5q), folks think `Drop` is better than `None` as it gives clarity by explicitly communicating the intention.

I suggest that we move forward and merge the PR before Stable release.

**Review by MrAlias — APPROVED** [review]

**Review by jonatan-ivanov — APPROVED** [review]
+1 for Drop: dropping values has different meaning than not aggregating them.

**Review by carlosalberto — APPROVED** [review]

File: specification/metrics/sdk.md [hunk 1]
```diff
@@ -301,16 +301,16 @@ The SDK MUST provide the following `Aggregation` to support the
...
-- [None](./sdk.md#none-aggregation)
+- [Drop](./sdk.md#drop-aggregation)
 - [Default](./sdk.md#default-aggregation)
 - [Sum](./sdk.md#sum-aggregation)
 - [Last Value](./sdk.md#last-value-aggregation)
 - [Histogram](./sdk.md#histogram-aggregation)
 - [Explicit Bucket Histogram](./sdk.md#explicit-bucket-histogram-aggregation)
 
-#### None Aggregation
+#### Drop Aggregation
```
Comment (arminru) [inline specification/metrics/sdk.md:311]: Please update the CHANGELOG accordingly.

**Review by arminru — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]
Approve pending Armin's comment

File: specification/metrics/sdk.md (hunk 1 above)
Comment (cijothomas) [reply specification/metrics/sdk.md:311]: done.

**Pull request merged**